src/coreComponents/fileIO/coupling/ChomboCoupler.cpp

```diff
@@ -159,8 +160,10 @@ void ChomboCoupler::copyNodalData()
   NodeManager const & nodes = m_mesh.getNodeManager();
   localIndex const numNodes = nodes.size();
   arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const & referencePos = nodes.referencePosition();
-  arrayView2d< real64 const, nodes::TOTAL_DISPLACEMENT_USD > const & displacement = nodes.totalDisplacement();
-  arrayView2d< real64 const, nodes::TOTAL_DISPLACEMENT_USD > const & velocity = nodes.velocity();
+  arrayView2d< real64 const, nodes::TOTAL_DISPLACEMENT_USD > const & displacement =
```

Comment:
```suggestion
  solidMechanics::array2dLayoutTotalDisplacementConst const & displacement =
```
Not that this is any more compact, but could we do something like this to make the LHS easier to type? `auto` doesn't work because we have to specify the `const` inside the template template argument.

Reply:
Done, now in this PR, the accessor calls look like:
```
using namespace extrinsicMeshData;

solidMechanics::arrayViewConst2dLayoutTotalDisplacement const & u =
  nodeManager.getExtrinsicData< solidMechanics::totalDisplacement >();
```